deploy/kubernetes/overlays/dev/kustomization.yaml

```diff
@@ -9,14 +9,6 @@ resources:
 # Here dev overlay is using the same image as alpha
 transformers:
 - ../../images/stable-master
-# Apply patches to support dynamic provisioning for hyperdisks
```

Comment:
Why is this / driver-args.yaml being deleted?

Reply:
There shouldn't be a need for the driver-args as the command line arguments are in the base controller.yaml.